tests/unit/other.tcl

```diff
@@ -438,6 +438,11 @@ start_cluster 1 0 {tags {"other external:skip cluster slow"}} {
         # disable resizing
         r config set rdb-key-save-delay 10000000
         r bgsave
+        wait_for_condition 1000 10 {
+            [s rdb_bgsave_in_progress] eq 1
+        } else {
+            fail "bgsave did not start in time"
+        }
```

Comment:
why would bgsave not start immediately in this case?
we just did a flush (foreground save), and added 128 keys..

Reply:
i am not sure...
i just want to eliminate all possible races. ie. make sure we have started the save

Reply:
ok, i had removed it.